request_a_govuk_domain/request/templates/registrar_details.html

```diff
@@ -12,28 +12,44 @@
 {% endblock %}
 
 {% block content %}
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            {% error_summary form %}
-            <h1 class="govuk-heading-xl">
-                Registrar details
-            </h1>
-
-            <p class="govuk-body">
-                You must be listed as a .gov.uk Approved Registrar on <a class="govuk-link"
-                                                                         href="https://registrars.nominet.uk/registry/dot-gov-uk/registrars/"
-                                                                         target="_blank">Nominet's website (opens in new
-                tab)</a>,
-                otherwise you cannot use this service.
-            </p>
-
-            <p class="govuk-body">
-                Email <code>support@domains.gov.uk</code> for more information
-            </p>
-
-            {% crispy form %}
-        </div>
+{% load custom_filters %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-notification-banner govuk-notification-banner--blue" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+          <div class="govuk-notification-banner__header">
+          <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+              Important
+          </h2>
+          </div>
+          <div class="govuk-notification-banner__content">
+              <h2 class="govuk-heading-s">Your session can time out</h2>
```

Comment:
Please make sure your indentation is correct everywhere once and for all, so I don't have to remind you multiple times.